classes/judge.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, data=None):
         self.type = "judge"
         for property in Property.all_properties:
             self.add_property(property, data)
-        self.remaining = int(self.properties.get("commitment", 50))
+        self.remaining = int(self.properties.get("commitment", 50) or 0) 
```

Comment:
What is the `or 0` doing?

Reply:
I had a failure where it tried to make "" into an int. This is a quick way to hack around that.

Reply:
Looks like the generator script is broken.  I'll get that fixed.